src/ppx_deriving.ml

```diff
@@ -2,6 +2,7 @@ open Longident
 open Location
 open Asttypes
 open Parsetree
+open Ast_mapper
```

Comment:
You don't have to open the Ast_mapper, all references are qualified

On the other hand you can even remove Ast_helper below